UefiCpuPkg/PiSmmCpuDxeSmm/SmmProfileInternal.h

```diff
@@ -27,7 +27,7 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 
 #define PEBS_RECORD_NUMBER  0x2
 
-#define MAX_PF_ENTRY_COUNT  10
+#define MAX_PF_ENTRY_COUNT  100

```

Comment:
Can this macro change fix all potential issues?
what's the value of R8? Will it be a very very big value that even 100 cannot work?

Reply:
this is due to the NV cache Buffer copy, the page of NV cache Buffer is 0x3E, it will creat the 60+ entry in the table. But i just aware the value is platform related. So, it's not the genernal/common fix.

PeiVariable: NV cache Buffer is: 0xDF2D4000, number of pages is: 0x3E

Reply:
remove the above change